cmd/fake-cmd-server/fake-cmd-server.go

```diff
@@ -16,9 +16,6 @@ import (
 )
 
 func main() {
-	// set new default verbosity, was set to 0 by glog
-	goflag.Set("v", "2")
```

Comment:
Asking to understand, does kubevirt.io/client-go/log set verbosity level to 2 by default?